Chead: head recursive fibonacci for C++ and C

Villiamfj opened the pull request:
Chead was just a funny name.

These are not from rosetta code

Fixed an error in bench_func.sh, that overwrites the testName variable.

Villiamfj committed:
c fib head og cpp fib head

Villiamfj committed:
Removed debuggin

Villiamfj committed:
Comments on where the code is from

Villiamfj commented on benchmarks/fibsequence/cpphead/bench.cpp:
```diff
@@ -0,0 +1,37 @@
+}
+
+// test method (our own implementation)
+unsigned long long int fibonacci(unsigned long long int n) {
```
Jeg er lidt i tvivl om typen på den her
Rosetta brugte bare unsigned int, dog i c var det brugt long long int

sabj17 replied:
Vi skal også bruge unsigned til C. Unsigned overflow er "well-defined" - aka den laver bare et wrap-around. Hvor signed int overflow er undefined

sabj17 replied:
Plus, unsigned kan indeholde en større værdi

Villiamfj replied:
jeg har lavet dem om til unsigned

Villiamfj committed:
converted chead and cpp head to unsigned int

cpp was unsigned long long int

sabj17 reviewed (approved):
Seems okay. But does it run?

Villiamfj committed:
ups, two prints in bench.c

Villiamfj committed:
Revert "ups, two prints in bench.c"

This reverts commit 6eba65a3dbf93141412e8f160ca7e83251926476.

Villiamfj committed:
ups, two prints in bench.c (fixed)

Villiamfj committed:
fixed naming error in bench_func.sh

bench_func overrites the variable "testName" in the benchRunners.
this is a problem for tests like chead.
